### PR TITLE
Implement Datadog Monitors for insufficient of free swap space

### PR DESCRIPTION
**Pull request by SweetOps**
## What

* Implement Datadog Monitors for insufficient of free swap space

## Why

* Necessary to have module for creation datadog alerts

### CODE REVIEW

File: swap/swap.tf [hunk 1]
```diff
@@ -0,0 +1,30 @@
+resource "datadog_monitor" "swap_free" {
+  count = "${var.monitor_enabled ? 1 : 0}"
+  name  = "Insufficient of free swap space ${module.label.id}"
+  type  = "${var.alert_type}"
+
+  message = <<EOF
+  Insufficient of free swap space  ${var.period} on host {host.name} ({host.ip})
```
Comment (osterman) [inline swap/swap.tf]: fix grammar

File: modules/swap/swap.tf [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  name  = "Insufficient of free swap ${module.label.id}"
+  type  = "${var.alert_type}"
+
+  message = <<EOF
+  Insufficient of free swap ${var.period} on host {host.name} ({host.ip})
+  ${var.remediation}
+  ${var.notify}
+  EOF
+
+  escalation_message = "Insufficient of free swap on host {host.name} ({host.ip}) hasn't been solved ${var.escalation_notify}"
```
Comment (osterman) [inline modules/swap/swap.tf]: Incorrect grammar

File: swap/swap.tf (hunk 1 above)
Comment (osterman) [reply swap/swap.tf]: remove extra ` `

File: modules/swap/swap.tf (hunk 2 above)
Comment (osterman) [reply modules/swap/swap.tf]: remove ` of `

File: modules/swap/main.tf [hunk 3]
```diff
@@ -0,0 +1,14 @@
+module "label" {
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.2.1"
```
Comment (osterman) [inline modules/swap/main.tf]: Upgrade label module

File: modules/swap/swap.tf [hunk 4]
```diff
@@ -0,0 +1,30 @@
+resource "datadog_monitor" "swap_free" {
+  count = "${var.monitor_enabled ? 1 : 0}"
+  name  = "Insufficient of free swap ${module.label.id}"
```
Comment (osterman) [inline modules/swap/swap.tf]: See: https://github.com/cloudposse/terraform-datadog-monitor/pull/2/files#r145316623

File: modules/swap/swap.tf [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  type  = "${var.alert_type}"
+
+  message = <<EOF
```
Comment (osterman) [inline modules/swap/swap.tf]: What's the behavior of TF HEREDOCs? Will it strip off these leading spaces? If not, we should consider moving this to a template file.

**Comment by aknysh** [comment]
closed in favor of https://github.com/cloudposse/terraform-datadog-monitor/pull/8

**Pull request closed without merging**